SentryTestUtils/TestClient.swift

```diff
@@ -0,0 +1,183 @@
+import Foundation
```

Comment:
Nothing actually changed in this file besides marking things `public`. It just didn't diff well with its rename.